### DISCUSSION TITLE
Calendar Escape Key Error v10.8.3

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#2789] [PrimeReact] [answered]
We just updated to PrimeReact v10.8.3. The update created a glitch in our two calendar components on the page in our software to create an appointment, i.e., start date/time calendar and end date/time calendar.

We assume this is related to [Fix #7008](https://github.com/primefaces/primereact/pull/7008)

The error occurs when a user is selecting a date/time in the calendar overlay pop-up, then while the overlay pop-up is still open, they click on the other calendar. Instead of the current overlay closing and the new overlay opening, this error occurs:

![calendar-error](https://github.com/user-attachments/assets/444e6b9b-c450-418a-a482-6bf3f004132a)

**Comment by melloware** [comment, marked as answer]
Already fixed for 10.8.4: https://github.com/primefaces/primereact/issues/7220
